BUILDING.md

```diff
@@ -63,6 +63,30 @@ E.g. [configuration-cache support](https://github.com/elastic/elasticsearch/issu
 There are a few guidelines to follow that should make your life easier to make changes to the elasticsearch build.
 Please add a member of the `es-delivery` team as a reviewer if you're making non-trivial changes to the build.
 
+#### Adding or updating a dependency
+
+We rely on [Gradle dependency verification](https://docs.gradle.org/current/userguide/dependency_verification.html) to mitigate the security risks and avoid integrating compromised dependencies.
+
+This requires to have third party dependencies and there checksums listed in `gradle/verification-metadata.xml`.
```

Comment:
Oh god why is it XML?

Reply:
don't get me started :) unfortunaly there's no other option.